linera-sdk/src/contract/test_runtime.rs

```diff
@@ -896,6 +896,22 @@ where
     pub fn validation_round(&mut self) -> Option<u32> {
         self.round
     }
+
+    /// TODO: What is this?
+    pub fn microchain_start(&mut self, _chain_state: Vec<u8>) -> (Vec<u8>, Vec<u8>, Vec<u8>) {
+        todo!()
+    }
+
+    /// TODO: What is this?
+    pub fn microchain_transition(
+        &mut self,
+        _chain_proof_hash: DataBlobHash,
+        _chain_proofs: Vec<u8>,
+        _chain_state: Vec<u8>,
+        _zstore_view: Vec<u8>,
+    ) -> (Vec<u8>, Vec<u8>, Vec<u8>) {
+        todo!()
+    }
```

Comment:
Do we need these functions or are we gonna fill them out eventually? If nothing breaks, I would just remove them (though if it's needed to be present for the test WASM runtime, we can leave it as-is for now)